src/java.management.rmi/share/classes/javax/management/remote/rmi/package-info.java

```diff
@@ -0,0 +1,326 @@
+ *
+ *
+ *    <h3>Connector server attributes</h3>
+ *
+ *    <p>When using the default JRMP transport, RMI socket factories can
+ *      be specified using the attributes
+ *      <code>jmx.remote.rmi.client.socket.factory</code> and
+ *      <code>jmx.remote.rmi.server.socket.factory</code> in the
+ *      <code>environment</code> given to the
+ *      <code>RMIConnectorServer</code> constructor.  The values of these
+ *      attributes must be of type {@link
+ *      java.rmi.server.RMIClientSocketFactory} and {@link
+ *      java.rmi.server.RMIServerSocketFactory}, respectively.  These
+ *      factories are used when creating the RMI objects associated with
+ *      the connector.</p>
+ *
+ *    <h2>Creating an RMI connector client</h2>
+ *
+ *    <p>An RMI connector client is usually constructed using {@link
+ *      javax.management.remote.JMXConnectorFactory}, with a
+ *      <code>JMXServiceURL</code> that has <code>rmi</code> as its protocol.</p>
+ *
+ *    <p>If the <code>JMXServiceURL</code> was generated by the server,
+ *      as described above under <a href="#servergen">"connector
+ *      addresses generated by the server"</a>, then the client will
+ *      need to obtain it directly or indirectly from the server.
+ *      Typically, the server makes the <code>JMXServiceURL</code>
+ *      available by storing it in a file or a lookup service.</p>
+ *
+ *    <p>If the <code>JMXServiceURL</code> uses the directory syntax, as
+ *      described above under <a href="#directory">"connector addresses
+ *      based on directory entries"</a>, then the client may obtain it
+ *      as just explained, or client and server may both know the
+ *      appropriate directory entry to use.  For example, if the
+ *      connector server for the Whatsit agent uses the entry
+ *      <code>whatsit-agent-connector</code> in the RMI registry on host
+ *      <code>myhost</code>, then client and server can both know
+ *      that the appropriate <code>JMXServiceURL</code> is:</p>
+ *
+ *    <pre>
+ *    <code>service:jmx:rmi:///jndi/rmi://myhost/whatsit-agent-connector</code>
+ *    </pre>
+ *
+ *    <p>If you have an RMI stub of type {@link
+ *      javax.management.remote.rmi.RMIServer RMIServer}, you can
+ *      construct an RMI connection directly by using the appropriate
+ *      constructor of {@link javax.management.remote.rmi.RMIConnector
+ *      RMIConnector}.</p>
+ *
+ *    <h2>Dynamic code downloading</h2>
+ *
+ *    <p>If an RMI connector client or server receives from its peer an
+ *      instance of a class that it does not know, and if dynamic code
+ *      downloading is active for the RMI connection, then the class can
+ *      be downloaded from a codebase specified by the peer.
+ *      {@extLink rmi_guide Java RMI Guide} explains this in more detail.</p>
+ *
+ *    @see <a href="http://www.ietf.org/rfc/rfc2045.txt">RFC 2045,
+ *    section 6.8, "Base64 Content-Transfer-Encoding"</a>
+ *
+ *
+ *    @since 1.5
+ *
+ **/
```

Comment:
Extra *  `*/` is sufficient.

Reply:
Fixed in [393bf3a](https://github.com/openjdk/jdk/pull/19263/commits/393bf3a36e3fb0811f89ef21cbdbb4a32a7a3e21), thanks.